docs/builder-single-focus-redesign/mockups/single-focus-immersive.html

```
@@ -0,0 +1,140 @@
+<h2>Single-focus — immersive, no wrapping card</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)